src/components/Filters/spec.js

```diff
@@ -133,6 +164,34 @@ describe('Filters component', () => {
       expect(component.find(FiltersSummary)).toHaveLength(0)
       component.unmount()
     })
+
+    it('should filter using "is" operator for both key and value properties of an object', () => {
```

Comment:
Contrary to the test title, this test doesn't actually check that the filtering behaves as expected, it just checks that the label is present - can you add a test for schemasieve that asserts that the filtering behaves as expected?